testSpace/myTest.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python2
+
+import csv
+import sys
+sys.path.append("/home/ltsai/Work/github/ntuhep/bpkUtility")
+
+from python.analysis import nameDict
+'''
+lst = [2,3,4]
+
+for index,element in enumerate(lst):
+    print "index",index,"element",element
+'''
+branchName=[
+    'EvtInfoBranches',
+    'GenInfoBranches',
+    'JetInfoBranches',
+    'LepInfoBranches',
+    'PhotonInfoBranches',
+    'TrgInfoBranches',
+    'VertexInfoBranches' ]
+
+diffNum={}
+diffName={}
+for it in range( len(branchName) ):
```

Comment:
Could you try using the python-like for loop? for example:
```
for branch in branchName:
```
as the index `it` was never used anywhere other than branchName[it]